msvc/mono.def

```diff
@@ -1,6 +1,4 @@
 ; file generated by create-windef.pl
-EXPORTS
```

Comment:
And since the .def file has been manually edited since the change in the script (the script has not been rerun), this problem has not submerged until now. I can see submit a new PR that fix he windows build issue + the script.